components/Alerts/alerts.css

```diff
@@ -0,0 +1,38 @@
+@import url(./root.css);
+
+.alert{
+        padding: 1rem;
+    font-size: 1.5rem;
+    background-color: var(--PURE-BLACK);
+    color: var(--PURE-WHITE);
+    border-radius: 4px;
+    width: 50vw;
+    word-spacing:6px;
+}
+.alert.error{
+background-color: var(--ERROR-RED);
```

Comment:
indentation can be improved.

Reply:
Thanks for reviewing Neha I will do that . :)